README.md

```diff
@@ -109,6 +150,9 @@ See the [Usage Guide](#usage-guide) for more information on how to get started u
 
 The Okta [Identity Engine][] (OIE) is a platform service that allows enterprises to build more flexible access experiences that are tailored to their organizational needs. The Okta Sign-in Widget supports OIE in all [usage](#usage-guide) scenarios.
 
+> **Note**:
+Unless otherwise noted, this README assumes you are using [Identity Engine][]. Information on using the widget with the "classic" engine can be found in [this document](https://github.com/okta/okta-signin-widget/blob/master/docs/classic.md)
```

Comment:
We have
- "classic" engine
- "classic engine"
- Classic Engine

We should try to be consistent in both docs about the name. 
I think we are ok for Identity Engine, since as far as I can see we don't use double quotes anywhere for it